content/docs/latest/installing/vms/hyper-v.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Running Flatcar Container Linux on Hyper-V
+title: Running on Hyper-V
+weight: 30
+aliases:
+    - ../../os/booting-on-hyper-v
+    - ../../cloud-providers/booting-on-hyper-v
+---
+
+_While we always welcome community contributions and fixes, please note that Hyper-V is not an officially supported platform at this time. (See the [platform overview](/#installing-flatcar).)_
+
+These instructions will walk you through running Flatcar Container Linux on Hyper-V.
+
+## Choose a channel
+
+Flatcar Container Linux is designed to be updated automatically with different schedules per channel. You can [disable this feature][update-strategies], although we don't recommend it. Read the [release notes][release-notes] for specific features and bug fixes.
+
+## Creating a config-drive
+
+Cloud-config can be specified by attaching a [config-drive](https://github.com/flatcar/coreos-cloudinit/blob/master/Documentation/config-drive.md) with the label `config-2`. This is commonly done through whatever interface allows for attaching CD-ROMs or new drives.
+
+Note that the config-drive standard was originally an OpenStack feature, which is why you'll see strings containing `openstack`. This filepath needs to be retained, although Flatcar Container Linux supports config-drive on all platforms.
+
+For more information on customization that can be done with cloud-config, head on over to the [cloud-config guide](https://github.com/flatcar/coreos-cloudinit/blob/master/Documentation/cloud-config.md).
+
+You need a config-drive to configure at least one SSH key to access the virtual machine. If you are in hurry, you can create a basic config-drive with following steps:
+
+```powershell
+#ps1
+# mkisofs can be downloaded from https://github.com/cloudbase/cloudbase-init-test-resources/tree/master/bin
+# create an Openstack config drive folder structure
+mkdir config-drive-metadata/openstack/latest
+echo '{"hostname": "my_flatcar_01.local", "name": "my_flatcar_01", "public_keys": {"userkey": "INSERT_HERE_PUBLIC_SSH_KEY"}' > config-drive-metadata/openstack/latest/meta_data.json
+# create the config drive
+& "mkisofs.exe" -o "config-drive.iso" -ignore-error -ldots -allow-lowercase -allow-multidot -l -publisher "cbsl" -quiet -J -r -V "config-2" "config-drive-metadata"
```

Comment:
what does the `&` do?

Reply:
`&` is a construct to not run .\<binary_file_name>. I prefer it sometimes to be clearer.